src/components/Table/TableToolbar/TableToolbar.jsx

```diff
@@ -173,16 +184,21 @@ const TableToolbar = ({
         </TableBatchAction>
       ))}
     </StyledTableBatchActions>
-    {hasSearch ? (
-      <StyledToolbarSearch
-        {...search}
-        translateWithId={(...args) => tableTranslateWithId(i18n, ...args)}
-        id={`${tableId}-toolbar-search`}
-        onChange={event => onApplySearch(event.currentTarget ? event.currentTarget.value : '')}
-        disabled={isDisabled}
-      />
+    {hasRowCount ? (
+      <ToolBarResultLabel>
+        {i18n.rowCountLabel}: {totalItemsCount}
```

Comment:
We can't do this for labels it will ultimately run into BIDI issues and other potential formatting issues.  The rowCountLabel should be a label like `Count: {0}` and we transform the message + arg into a final string.  Not sure how the table handles this, so might require some research.

Reply:
Instead of a string, this can be a function that returns a string. A good example of this pattern is the [`itemRangeText` prop in the Carbon `<Pagination>`](https://github.com/carbon-design-system/carbon/blob/master/packages/react/src/components/Pagination/Pagination.js#L134)